### PR TITLE
[RW-1988][risk=no] Updating PPI tree to make Survey and Questions selectable

### PR DESCRIPTION
**Pull request by freemabd1**
Updating PPI tree to make Survey and Questions selectable

### CODE REVIEW

File: ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts [hunk 1]
```diff
@@ -103,7 +97,7 @@ export class QueryCohortDefinitionComponent implements OnInit {
       } else if (param.type === 'VISIT') {
         return {items: `${typeToTitle(_type)} | ${typeMatched.customString}`,
           type: param.type};
-      } else {
+      }  else {
```
Comment (dolbeew) [inline ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts]: remove extra space

File: ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts [hunk 2]
```diff
@@ -175,14 +136,18 @@ export class QueryCohortDefinitionComponent implements OnInit {
         return acc === '' ? `ATC | ${d.value}` : `${acc}, ${d.value}`;
       }
     } else if (d.type === 'PM' || d.type === 'VISIT') {
+
```
Comment (dolbeew) [inline ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts]: remove blank line

File: ui/src/app/cohort-search/node-info/node-info.component.ts [hunk 3]
```diff
@@ -113,8 +114,10 @@ export class NodeInfoComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   get selectAllChildren() {
+    // console.log(this.node.toJS());
```
Comment (dolbeew) [inline ui/src/app/cohort-search/node-info/node-info.component.ts]: remove comment

File: ui/src/app/cohort-search/node-info/node-info.component.ts [hunk 4]
```diff
@@ -145,8 +148,14 @@ export class NodeInfoComponent implements OnInit, OnDestroy, AfterViewInit {
        */
 
       if (this.selectAllChildren) {
+        // console.log(this.node);
```
Comment (dolbeew) [inline ui/src/app/cohort-search/node-info/node-info.component.ts]: remove comment

File: ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts (hunk 1 above)
Comment (freemabd1) [reply ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts]: Done.

File: ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts (hunk 2 above)
Comment (freemabd1) [reply ui/src/app/cohort-review/query-cohort-definition/query-cohort-definition.component.ts]: Done.

File: ui/src/app/cohort-search/node-info/node-info.component.ts (hunk 3 above)
Comment (freemabd1) [reply ui/src/app/cohort-search/node-info/node-info.component.ts]: Done.

File: ui/src/app/cohort-search/node-info/node-info.component.ts (hunk 4 above)
Comment (freemabd1) [reply ui/src/app/cohort-search/node-info/node-info.component.ts]: Done

**Pull request merged**